{"tag":"AnalysisSuccess","contents":{"requestedTools":null,"toolResults":[[{"tag":"Infer"},{"tag":"ToolFailure","contents":{"tag":"ToolSurfaceableFailure","contents":["BuildError while running Infer on master: Is the build broken?","[DEBUG] [28/May/2019:08:58:16 +0000] [()] Analysis is starting.\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Getting analysis results for commit \"master\"\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Reading config file\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Final, parsed, repository configuration structure:\n    ProjSet\n      { buildCommand = Nothing\n      , argsList = []\n      , interestingIssue = <function>\n      , setupScript = Nothing\n      , toolList = Nothing\n      , customTools = []\n      , whitelist = Nothing\n      , jdk11 = False\n      }\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Determining tools...\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  infer ENABLED\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  errorprone ENABLED\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  hlint DISABLED, not applicable\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] 2 applicable tools\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Running infer on branch master\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]   Running infer bulk with a non-explicit build system.\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Infer BuildError: Src branch: All build attempts failed: Capture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':EventBusPerformance'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all files for configuration ':EventBusPerformance:classpath'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not resolve com.android.tools.build:gradle:2.3.3.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :EventBusPerformance\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]       > Could not resolve com.android.tools.build:gradle:2.3.3.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/2.3.3/gradle-2.3.3.pom'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]             > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/2.3.3/gradle-2.3.3.pom'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]                > jcenter.bintray.com: Name or service not known\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace. \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 1s\\nERROR: couldn't run compilation command `[u'./gradlew', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-ed30e7ecc702234a --out\\n  /tmp/analyzing-ed30e7ecc702234a/infer-out -- ./gradlew assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\nCapture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n08:58:26.728 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.728 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file '/tmp/analyzing-ed30e7ecc702234a/build.gradle' line: 24\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'analyzing-ed30e7ecc702234a'.\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Cannot add task 'wrapper' as a task with that name already exists.\\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n08:58:26.731 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.736 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n08:58:26.737 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n08:58:26.737 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 8s\\nERROR: couldn't run compilation command `[u'gradle', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-ed30e7ecc702234a --out\\n  /tmp/analyzing-ed30e7ecc702234a/infer-out -- gradle assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\n\n"]}}],[{"tag":"ErrorProne"},{"tag":"ToolFailure","contents":{"tag":"ToolSurfaceableFailure","contents":["Execution Failed: ---STDOUT---\n---ERR---\n---CMD---\nCreateProcess {cmdspec = RawCommand \"gradle\" [\"-q\",\"-I\",\"/tmp/analyzing-ed30e7ecc702234a/muse-compdb.gradle\",\"assemble\"], cwd = Just \"/tmp/analyzing-ed30e7ecc702234a\", env = Nothing, std_in = Inherit, std_out = Inherit, std_err = Inherit, close_fds = False, create_group = False, delegate_ctlc = False, detach_console = False, create_new_console = False, new_session = False, child_group = Nothing, child_user = Nothing, use_process_jobs = False}","[DEBUG] [28/May/2019:08:58:16 +0000] [()] Analysis is starting.\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Getting analysis results for commit \"master\"\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Reading config file\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Final, parsed, repository configuration structure:\n    ProjSet\n      { buildCommand = Nothing\n      , argsList = []\n      , interestingIssue = <function>\n      , setupScript = Nothing\n      , toolList = Nothing\n      , customTools = []\n      , whitelist = Nothing\n      , jdk11 = False\n      }\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Determining tools...\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  infer ENABLED\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  errorprone ENABLED\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]  hlint DISABLED, not applicable\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] 2 applicable tools\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:08:58:16 +0000] [()] Running infer on branch master\n[DEBUG] [28/May/2019:08:58:16 +0000] [()]   Running infer bulk with a non-explicit build system.\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Infer BuildError: Src branch: All build attempts failed: Capture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n13:34:20.484 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred configuring project ':EventBusPerformance'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all files for configuration ':EventBusPerformance:classpath'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not resolve com.android.tools.build:gradle:2.3.3.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :EventBusPerformance\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]       > Could not resolve com.android.tools.build:gradle:2.3.3.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/2.3.3/gradle-2.3.3.pom'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]             > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/2.3.3/gradle-2.3.3.pom'.\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]                > jcenter.bintray.com: Name or service not known\\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.485 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace. \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n13:34:20.486 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 1s\\nERROR: couldn't run compilation command `[u'./gradlew', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-ed30e7ecc702234a --out\\n  /tmp/analyzing-ed30e7ecc702234a/infer-out -- ./gradlew assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\nCapture:\n-----\nCall Stack: \n------Execution Failed: \"Capturing in gradle mode...\\n08:58:26.728 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.728 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file '/tmp/analyzing-ed30e7ecc702234a/build.gradle' line: 24\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'analyzing-ed30e7ecc702234a'.\\n08:58:26.729 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Cannot add task 'wrapper' as a task with that name already exists.\\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:\\n08:58:26.730 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.\\n08:58:26.731 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] \\n08:58:26.736 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org\\n08:58:26.737 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] \\n08:58:26.737 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 8s\\nERROR: couldn't run compilation command `[u'gradle', '--debug', u'assemble']`\\nInternal Error:   /opt/infer-linux64-v0.16.0/lib/infer/infer/bin/../lib/python/infer.py\\n  --continue -j 2 --project-root /tmp/analyzing-ed30e7ecc702234a --out\\n  /tmp/analyzing-ed30e7ecc702234a/infer-out -- gradle assemble:\\n  exited with code 1\\nError backtrace:\\nRaised at file \\\"string.ml\\\", line 142, characters 16-31\\nCalled from file \\\"src/string.ml\\\", line 402, characters 12-33\\nCalled from file \\\"src/string.ml\\\", line 411, characters 11-33\\nRe-raised at file \\\"base/Die.ml\\\" (inlined), line 26, characters 8-56\\nCalled from file \\\"base/Die.ml\\\", line 32, characters 6-39\\nCalled from file \\\"integration/Driver.ml\\\", line 164, characters 2-16\\nCalled from file \\\"integration/Driver.ml\\\", line 282, characters 6-409\\nCalled from file \\\"integration/Driver.ml\\\", line 304, characters 2-29\\nCalled from file \\\"infer.ml\\\", line 20, characters 2-36\\nCalled from file \\\"infer.ml\\\", line 131, characters 8-54\\n\\nRun the command again with `--keep-going` to try and ignore this error.\\n\"\n\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Sanitizing repository\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Trying to autodetect the build system based on files present in the repository.\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Using build systems of [Gradlew [\"assemble\"],BuildGradle [\"assemble\"]] to build compilation database.\n[DEBUG] [28/May/2019:13:34:20 +0000] [()] Wrote the plugin for use with ./gradlew -I /tmp/analyzing-ed30e7ecc702234a/muse-compdb.gradle assemble\n[DEBUG] [28/May/2019:13:34:22 +0000] [()] Wrote the plugin for use with gradle -I /tmp/analyzing-ed30e7ecc702234a/muse-compdb.gradle assemble\n[DEBUG] [28/May/2019:13:34:24 +0000] [()] <function> BuildError: BuildError {renderError = \"Execution Failed: ---STDOUT---\\n---ERR---\\n---CMD---\\nCreateProcess {cmdspec = RawCommand \\\"gradle\\\" [\\\"-q\\\",\\\"-I\\\",\\\"/tmp/analyzing-ed30e7ecc702234a/muse-compdb.gradle\\\",\\\"assemble\\\"], cwd = Just \\\"/tmp/analyzing-ed30e7ecc702234a\\\", env = Nothing, std_in = Inherit, std_out = Inherit, std_err = Inherit, close_fds = False, create_group = False, delegate_ctlc = False, detach_console = False, create_new_console = False, new_session = False, child_group = Nothing, child_user = Nothing, use_process_jobs = False}\"}\n"]}}]]}}
